### PR TITLE
8253321: java.util.Locale.LanguageRange#equals is inconsistent after calling hashCode

### PR DESCRIPTION
**Pull request by naotoj**
Hi,

Please review the fix to JDK-8253321. As in the issue, uninitialized (cached) hash code was incorrectly referenced in equals() method. Removing it will correct the problem. Also, unrelated to the issue, I fixed a parameter description in a private method.

Naoto

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8253321](https://bugs.openjdk.java.net/browse/JDK-8253321): java.util.Locale.LanguageRange#equals is inconsistent after calling hashCode

### Reviewers
 * [Joe Wang](https://openjdk.java.net/census#joehw) (@JoeWang-Java - **Reviewer**)
 * [Roger Riggs](https://openjdk.java.net/census#rriggs) (@RogerRiggs - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/260/head:pull/260`
`$ git checkout pull/260`

### CODE REVIEW

File: test/jdk/java/util/Locale/LanguageRangeTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/**
+ * @test
+ * @bug 8253321
+ * @summary test LanguageRange class
+ * @run testng/othervm LanguageRangeTest
```
Comment (JoeWang-Java) [inline test/jdk/java/util/Locale/LanguageRangeTest.java:27]: Is it necessary to run in othervm?

**Comment by JoeWang-Java** [comment]
Also, copyright year may need to be updated to 2020 for Locale.java.

**Review by JoeWang-Java — APPROVED** [review]

File: test/jdk/java/util/Locale/LanguageRangeTest.java (hunk 1 above)
Comment (naotoj) [reply test/jdk/java/util/Locale/LanguageRangeTest.java]: Removed "othervm"

**Comment by naotoj** [comment]
> Also, copyright year may need to be updated to 2020 for Locale.java.

Thanks, Joe. Updated.

**Review by JoeWang-Java — APPROVED** [review]

**Review by RogerRiggs — APPROVED** [review]

**Pull request closed without merging**